shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/model/TableMetaData.java

```diff
@@ -47,10 +49,15 @@
     private final List<String> primaryKeyColumns = new ArrayList<>();
     
     public TableMetaData() {
-        this(Collections.emptyList(), Collections.emptyList());
+        this(null, Collections.emptyList(), Collections.emptyList());
```

Comment:
An empty string is better than null to avoid npe from my point.

Reply:
Yes, I agree.